docs/rootchainFunctions.md

```diff
@@ -0,0 +1,60 @@
+# PLASMA MVP Rootchain Documentation
+## RootChain.sol
+**function** `RootChain()` </br>
+Contract constructor that sets the validator of the child chain.
+
+**function** `submitBlock(bytes32 root)` </br>
+The validator submits the merkle root of a child chain block. </br>
+`@param bytes32 root`: merkle root of the child chain
+
+**function** `deposit(uint blocknum, bytes txBytes)` </br>
+Sender can deposit Eth into the smart contract, which will become redeemable on the child chain. </br>
+`@param uint blocknum`: the current child chain block number </br>
+`@param bytes txBytes`: the transaction bytes of the deposit, which is an encoded list of 15 elements.
+
+**function** `startExit(uint256[3] txPos, bytes txBytes, bytes proof, bytes sigs)` </br>
+Begins the exit procedure for exiting a utxo on the child chain. The function checks that the inputs are valid, that this exit hasn't been finalized or challenged before, and that the sender has bonded funds to this exit. Then, it adds the exit to the priority queue. </br>
+`@param uint256 txPos[0]`: plasma block number in which the transaction occured </br>
+`@param uint256 txPos[1]`: transaction Index within the block </br>
+`@param uint256 txPos[2]`: output Index within the transaction (either 0 or 1) </br>
+`@param bytes txBytes`: transaction bytes of the utxo </br>
+`@param bytes proof`: proof of transaction's existence in the child chain block </br>
```

Comment:
"Merkle proof of transaction's existence in the child chain block; should be 512-bytes long (the concatenation of 16 hashes, each 32 bytes long)"